layers/isis.go

```diff
@@ -0,0 +1,486 @@
+package layers
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"net"
+
+	"github.com/google/gopacket"
+)
+
+type ISISType byte
```

Comment:
It's better to use uint8 instead of byte type here

Byte types are mostly used, when working with string characters 
When you work with small integers use uint8. It adds clarity to your code